sources/platform/actors/running/usage_and_resources.md

```diff
@@ -13,25 +13,35 @@ slug: /actors/running/usage-and-resources
 
 ## Resources
 
-[Actors](../index.mdx) run in
-[Docker containers](https://www.docker.com/resources/what-container),
-which have a [limited amount of resources](https://phoenixnap.com/kb/docker-memory-and-cpu-limit) (memory, CPU, disk size, ...). When starting, the Actor needs to be allocated a certain share of those resources, such as how much CPU "horsepower" it will be able to use.
+[Actors](../index.mdx) run in [Docker containers](https://www.docker.com/resources/what-container), which have a [limited amount of resources](https://phoenixnap.com/kb/docker-memory-and-cpu-limit) (memory, CPU, disk size, etc). When starting, the Actor needs to be allocated a certain share of those resources, such as CPU capacity that is necessary for the Actor to run.
 
 ![Setting an Actor's memory](./images/usage_and_resources/memory-settings.png)
 
-By assigning an Actor's **memory** capacity, you also assign it a proportional amount of **CPU power**. In addition to CPU power, the memory allocation also influences disk size, which amounts to 2x the allocated memory.
+By assigning an Actor's **Memory** capacity, you also assign it a proportional amount of CPU power. The memory allocation also influences the disk size, which amounts to 2x the allocated memory.
 
-> [See the **Limits**](../../limits.md) page for detailed information on Actor memory, CPU limits, disk size and other limits.
+Check out the [Limits](../../limits.md) page for detailed information on Actor memory, CPU limits, disk size and other limits.
 
 ### Memory
 
-When invoking the Actor, the caller has to specify the amount of memory allocated for the Actor run. The amount of memory can be set to a power of 2 with a minimum of 128 MB, i.e., 256 MB, 512 MB, 1024 MB, 2048 MB, ..., 32768 MB.
+When invoking an Actor, the caller must specify the memomy allocation for the Actor run. The memory allocation must follow these requirements:
 
-Additionally, each user has a certain total limit of memory for running Actors. The sum of memory allocated for all running Actors and builds needs to fit into this limit, otherwise the user cannot start a new Actor. For more details, see [limits](../../limits.md).
+- It must be a power of 2.
+- The minimum allowed value is `128MB`
+- The maximum allowed value is `32768MB`
+- Acceptable values include: `128MB`, `256MB`, `512MB`, `1024MB`, `2048MB`, `4096MB`, `8192MB`, `16384MB`, and `32768MB`
```

Comment:
Suggestion: Maybe I'd just leave the listed values, and say it's powers of two from 128 to 32768. This way it's very verbose.

Reply:
To be honest I think being a bit more verbose here works as it clearly states all acceptable values.

Reply:
Ok 👍